testgrid/config.yaml

```diff
@@ -3770,8 +3772,8 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-10
   - name: ci-gce-kubeadm-upgrade-1.9-1.10
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-upgrade-1-9-1-10
-  - name: periodic-kubernetes-e2e-debs-pushed
-    test_group_name: periodic-kubernetes-e2e-debs-pushed
+  - name: periodic-kubernetes-e2e-packages-pushed
+    test_group_name: periodic-kubernetes-e2e-packages-pushed
```

Comment:
one question: should this stay in master-blocking dashboard, rather than branch-blocking ones?

Reply:
I added it there for more visibility.
It's good to have it failing more visibly than before so that people actually go triaging it IMO

Reply:
Is this job also testing package push for the release branches?

Reply:
No, it's just pulling and verifying

Reply:
ah I see the script covers last three releases, aka https://github.com/kubernetes/kubeadm/blob/master/tests/e2e/verify_packages_published.sh#L30